src/CrowdinApiClient/Api/BundleApi.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace CrowdinApiClient\Api;
+
+use CrowdinApiClient\Model\Bundle;
+use CrowdinApiClient\Model\File;
+use CrowdinApiClient\ModelCollection;
+
+/**
+ * Manage project bundles
+ *
+ * @package Crowdin\Api
+ */
+class BundleApi extends AbstractApi
+{
+    /**
+     * List Bundles
+     * @link https://support.crowdin.com/api/v2/#operation/api.projects.bundles.getMany API Documentation
```

Comment:
Please change these links to the `developer.crowdin.com`

See https://github.com/crowdin/crowdin-api-client-php/issues/99 for reference

Reply:
Done !